.circleci/config.yml

```diff
@@ -13,8 +13,8 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - v6-dependencies-{{ checksum "Pipfile.lock" }}-{{ checksum "thumbs/Pipfile.lock" }}
-            - v6-dependencies-default
+            - v7-dependencies-{{ checksum "Pipfile.lock" }}-{{ checksum "thumbs/Pipfile.lock" }}
+            - v7-dependencies-default
       # Ensure pip and pipenv are installed
```

Comment:
Deploys starting to fail on all branches bc of a stale dependency cache. Since this PR was already addressing CI fixes, I've added a commit to clear the cache and unblock deploys on master.